docs/guides/modules/orchestrate/pages/orchestration-cookbook.adoc

```diff
@@ -17,7 +17,7 @@ You may want to run a set of high credit-consuming tests only when a pull reques
 +
 .Add pipeline options for GitHub App
 image::guides:ROOT:add-pipeline.png[Add pipeline options for GitHub App]
```

Comment:
I think the image here is out of date. And to be honest, I don't think this image is needed so we can just drop it IMO
<img width="1017" height="822" alt="Screenshot 2025-09-22 at 11 35 00" src="https://github.com/user-attachments/assets/5a6c4b9b-1062-4c35-9c28-37f12a1eefdf" />